src/d3a/d3a_core/live_events.py

```diff
@@ -50,14 +50,15 @@ def __init__(self, area_uuid, area_params):
     def apply(self, area):
         if area.uuid != self.area_uuid:
             return False
-        self.sanitize_live_event_paramters_for_strategy_update()
         area_type = self.area_params.pop("type", None)
         if area_type is not None:
+            self.sanitize_live_event_paramters_for_strategy_update()
             if area.strategy is None:
                 return False
             if area_type == "MarketMaker":
                 area.strategy = MarketMakerStrategy(**self.area_params)
             elif area_type == "InfiniteBus":
+                self.area_params['energy_sell_rate'] = self.area_params.pop('energy_rate', None)
```

Comment:
You can also do the conversion on the d3a-web side in order to be more consistent with the other parameters. Leave it here for now, just leave a TODO comment to move it to d3a-web later.